lib/vdsm/virt/thinp.py

```diff
@@ -305,6 +305,22 @@ def _monitored_volumes(self):
                 if drive.needs_monitoring()]
 
     def _should_extend_volume(self, drive, volumeID, block_info):
+        if block_info.physical >= drive.getMaxVolumeSize(block_info.capacity):
+            # The volume was extended to the maximum size. physical may be
```

Comment:
"physical" -> "Physical" (type already exists there, so probably fix it in another patch)

Reply:
This is existing issue, not introduced by this PR, but I can fix it while moving the code.

Reply:
Here is master code with this typo:

```
330         if block_info.physical >= drive.getMaxVolumeSize(block_info.capacity):
331             # The volume was extended to the maximum size. physical may be
332             # larger than maximum volume size since it is rounded up to the
333             # next lvm extent.
334             return False
```

I move this code in later, so I will fix it this patch:
https://github.com/oVirt/vdsm/pull/124/commits/5e65bfea2b884aea7a8872ab5fc4e7ae0e1bfc6b